app.config.ts

```diff
@@ -20,22 +20,14 @@ const expoConfig: ExpoConfig = {
   name: 'Taito Template',
   scheme: config.scheme,
   owner: 'taito-united',
-  version: '1.0.0',
+  version: '0.0.1',
```

Comment:
![image](https://github.com/user-attachments/assets/3e943f36-1fdd-4d1b-8f9a-313505035045)

Reply:
The `package.json` has it as 0.0.1 and I figured an app created from the template shouldn't start at 1.0.0. 

Even **React Native** is still not there... 😀